app/views/issues/_form.html.erb

```diff
@@ -23,7 +23,28 @@
   <%= render 'tag_input', f: f %>
 
   <div class="form-actions">
-    <%= f.button :submit, nil, class: 'btn btn-primary' %> or
+    <div class="btn-group dropdown state-menu" data-behavior="state-menu">
```

Comment:
We're already using a partial in L23, probably that's a good idea here.

Maybe time to organise both inside a ./form/ folder as the're not used elsewhere? Not too fussed about this though.